docs/guides/getting-started-java-spring.md

```diff
@@ -81,7 +81,7 @@ To deploy your process, take the following steps:
 3. Change the **Cluster endpoint** to `http://localhost:26500/`, with no authentication.
 4. Click **Deploy**.
 
-When you open Operate at http://localhost:8081/, you should now note the process deployed to your local Self-Managed setup.
+When you open Operate at http://localhost:8080/operate/, you should now note the process deployed to your local Self-Managed setup.
```

Comment:
This URL changed when we switched to the plain-java installation.

Note that I've only added this change to vNext, because vCurrent still uses the docker-compose installation method.

Reply:
We would also need to change the tip admonition in Step 1, but it currently states that Operate can be found at `localhost:8080` (which redirects to `8080/operate/`).

Reply:
I said this poorly - I just expect them to match. Whether they use redirects or not, I don't care.

Reply:
I'll get them aligned.

Reply:
aligned in latest commit.